### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by Fedoto4kin** [openfoodfacts/openfoodfacts-laravel#4]
Hi, there is one issue when installed

laravel 8.0
guzzle 7.2

```
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Can only install one of: guzzlehttp/guzzle[6.5.x-dev, 7.2.0].
    - Can only install one of: guzzlehttp/guzzle[7.2.0, 6.5.x-dev].
    - Can only install one of: guzzlehttp/guzzle[6.5.x-dev, 7.2.0].
    - openfoodfacts/openfoodfacts-php 0.2.3 requires guzzlehttp/guzzle ^6.3 -> satisfiable by guzzlehttp/guzzle[6.5.x-dev].
    - openfoodfacts/openfoodfacts-laravel v0.2 requires openfoodfacts/openfoodfacts-php ^0.2.3 -> satisfiable by openfoodfacts/openfoodfacts-php[0.2.3].
    - Installation request for openfoodfacts/openfoodfacts-laravel ^0.2.0 -> satisfiable by openfoodfacts/openfoodfacts-laravel[v0.2].
    - Installation request for guzzlehttp/guzzle (locked at 7.2.0, required as ^7.0.1) -> satisfiable by guzzlehttp/guzzle[7.2.0].
```

**Comment by Fedoto4kin** [comment]
**[UPDATE]**
**Solved** by removing installed guzzle:
`composer remove guzzlehttp/guzzle`
And after that install as usual

**Comment by epalmans** [comment]
thanks @Fedoto4kin 
The upstream package does not yet support Guzzle 7.x, which is indeed being required by default on a freshly installed laravel 8 app.

PR'ed on openfoodfacts/openfoodfacts-php: https://github.com/openfoodfacts/openfoodfacts-php/pull/26

**Comment by Fedoto4kin** [comment]
> thanks @Fedoto4kin

You're welcome
> The upstream package does not yet support Guzzle 7.x, which is indeed being required by default on a freshly installed laravel 8 app.
> 
> PR'ed on openfoodfacts/openfoodfacts-php: [openfoodfacts/openfoodfacts-php#26](https://github.com/openfoodfacts/openfoodfacts-php/pull/26)

I see, thank you.

**Comment by epalmans** [comment]
Because of a dependency update (https://github.com/openfoodfacts/openfoodfacts-php/pull/28), this package is now capable of supporting Laravel 8. Available in v0.2.1
